.gitignore

```diff
@@ -106,3 +106,5 @@ dist
 .jekyll-cache
 .idea
 .vscode
+package-lock.json
+yarn.lock
```

Comment:
The lockfiles need to be committed. [Read more](https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files)

Reply:
Oh, I didn't know it, thanks for clearing things out 👌